tcan_can_j1939/include/tcan_can_j1939/J1939CanMsg.hpp

```diff
@@ -36,6 +38,25 @@ class J1939CanMsg : public tcan_can::CanMsg {
 
     constexpr uint32_t getParameterGroupCanId() const { return getCobId() & CAN_ID_PGN_MASK; }
 
+    friend std::ostream& operator<<(std::ostream& os, const J1939CanMsg& msg) {
+        os << "Priority: " << +msg.getPriority() << std::endl
```

Comment:
I haven't see this before with the prefix `+`. How is that used?

Reply:
A unary plus that promotes the uint8 to an int (a uint8_t is otherwise printed as a char). Used incorrectly below, which I just corrected.